spec/features/user_views_an_item_type_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
```

Comment:
Why separate these by 0 item types, + item types, then have a separate file and spec for one item type?